stable/mysql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mysql
-version: 0.3.7
+version: 0.3.8
```

Comment:
You cna bump the minor of the version, as this adds new functionality.

Reply:
done